forest/cli/tests/send_tests.rs

```diff
@@ -0,0 +1,120 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+#[cfg(test)]
+mod tests {
+    use std::str::FromStr;
+
+    use forest_cli::cli::{
+        send_cmd::FILAmount,
+        wallet_cmd::{format_balance_string, FormattingMode},
+    };
+    use fvm_shared::econ::TokenAmount;
+    use quickcheck_macros::quickcheck;
+
+    #[test]
+    fn invalid_attofil_amount() {
+        //attoFIL with fractional value fails (fractional FIL values allowed)
+        let amount = "1.234attofil";
+        assert!(FILAmount::from_str(amount).is_err());
+    }
+
+    #[test]
+    fn valid_attofil_amount() {
```

Comment:
Let's also test if `1234 afil` and `1234 a fil` parse.

Reply:
Oh, and also just `1234 a`.

Reply:
By looking at your code, I would have guessed all three of them to be valid.

Reply:
The initial spec for my code was to attempt to align with the code/behavior in Lotus as closely as possible, but that changed during development, and the code is still a WIP. So the question now is: what behavior do we want from Forest?

Reply:
What do you think about `a` or `a fil`?

Reply:
I think both are fine.